components/board/genericlinux/data.go

```diff
@@ -14,38 +14,6 @@ import (
 
 // adapted from https://github.com/NVIDIA/jetson-gpio (MIT License)
```

Comment:
Unrelated to the current PR: here's the comment explaining where any of this came from in the first place! I've been looking for this history. :upside_down_face: